lib/ws-kernel-picker.js

```diff
@@ -372,6 +373,7 @@ export default class WSKernelPicker {
       store.grammar,
       session
     );
-    this._onChosen(kernel);
+    const hkernel = new HKernel(kernel);
```

Comment:
I don't think this the right place to do this. `ws-kernel-picker` and `WSKernel` shouldn't know about `Kernel(=HKernel)`. The callback should deal with the conversion to Kernel(=HKernel).